docs/examples.md

```diff
@@ -86,6 +86,59 @@ new FlowJobBuilder(
 }
 ```
 
+## Pipeline Builder
+
+```groovy
+import jenkins.automation.builders.PipelineJobBuilder
+
+def script = """
+             node {
+                stage('First stage') {
+                    build job: 'Job 1',
+                    parameters: [
+                        [\$class: 'StringParameterValue', name: 'foo', value: 'bar'],
+                    ]
+                }
+            }
+"""
+
+def pipelineJob = new PipelineJobBuilder(
+    name: 'Pipeline builder',
+    description: 'This is a simple pipeline job',
+    pipelineScript: script,
+    emails: ['jane@example.com', 'joe@example.com']
+
+).build(this);
+
+pipelineJob.with {
+    logRotator {
+        numToKeep(365)
+    }
+}
+
+def pipelineJobStages = new PipelineJobBuilder(
```

Comment:
If we're going to support this `stages` stuff then somewhere, it needs to be made clear why you'd choose this over just using pipelineScript. If I'm coming into this Builder documentation and I see two ways to do the same thing, with no explanation of the difference or why I'd choose one over the other, I'm just confused